pdns/arguments.hh

```diff
@@ -106,9 +106,11 @@ public:
   void setCmd(const string &, const string &); //!< Add a command flag
   string &setSwitch(const string &, const string &); //!< Add a switch flag
   string helpstring(string prefix=""); //!< generates the --help
-  string configstring(bool current=false); //!< generates the --mkconfig
+  string configstring(bool current=false, bool full=true); //!< generates the --config
```

Comment:
methods with multiple default boolean args are weird; should one of them maybe not have a default arg or this be split into two (perhaps three) methods?

Reply:
I also wondered about that, perhaps it would be best to not give them defaults at all?

Reply:
the defaults are gone